pkg/util/machines/machines.go

```diff
@@ -33,12 +33,12 @@ func IsMachineHealthy(c client.Client, machine *mapiv1.Machine) bool {
 
 	readyCond := conditions.GetNodeCondition(node, v1.NodeReady)
 	if readyCond == nil {
-		glog.Infof("node %s does have 'Ready' condition", machine.Name)
+		glog.V(4).Infof("node %s does have 'Ready' condition", machine.Name)
 		return false
 	}
 
 	if readyCond.Status != v1.ConditionTrue {
-		glog.Infof("node %s does have has 'Ready' condition with the status %s", machine.Name, readyCond.Status)
+		glog.V(4).Infof("node %s does have has 'Ready' condition with the status %s", machine.Name, readyCond.Status)
```

Comment:
this does not seems related to the title of this commit

Reply:
done